source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: "WeatherFlow"
+description: "WeatherFlow weather sensor"
+ha_release: "0.38"
+ha_category: 
+  - Sensor
+ha_platforms:
+ - sensor
+ha_iot_class: "Local Polling"
+ha_quality_scale: silver
+ha_config_flow: true
+ha_codeowners:
+  - '@natekspencer'
+  - '@jeeftor'
+ha_domain: weatherflow
+---
+
+
+### WeatherFlow 
+
+The WeatherFlow integration uses the local UDP interface on a Weatherflow sensor, such as the Tempest, to discover and display weather data gathered vial the local network.
+
+### Installation Notes
+
+When you add this integration it may take a few minutes for all the entities to show up. The integration will listen for WeatherFlow UDP packets over the network and decode them as they arrive. As sensor-specific packets show up the sensor entity will be created.
```

Comment:
It might be a good idea to add this as a final step in the config flow, so user are aware when adding (and not afterward when they are seeking for help/debugging)/